src/txmempool.cpp

```diff
@@ -890,6 +895,18 @@ void CTxMemPool::ClearPrioritisation(const uint256& hash)
     mapDeltas.erase(hash);
 }
 
+std::map<uint256, std::pair<CAmount, bool>> CTxMemPool::GetPrioritisationMap() const
+{
+    AssertLockNotHeld(cs);
+    LOCK(cs);
+    std::map<uint256, std::pair<CAmount, bool>> result;
+    for (const auto& [txid, delta] : mapDeltas) {
+        const bool in_mempool{mapTx.count(txid) != 0};
```

Comment:
You're already looking up the tx here, so you could use `find()` instead and on success also return the total modified fee.

Reply:
Done, added a `std::optional<CAmount> modified_fee` to the `delta_info` struct